bindings/protocols/profinet/index.html

```diff
@@ -0,0 +1,1040 @@
+                <li><code>{api}</code> known as Application Process Identifier, for distingushing profiles parameter. See <a href="#vocabulary">vocabulary</a></li>
+                <li><code>{index}</code> identifers a parameter with slot/subslot address. See <a href="#vocabulary">vocabulary</a></li>
+                <li><code>{datalength}</code> defines the length of byte resource payload is expected to be. See <a href="#vocabulary">vocabulary</a></li>
+
+            </ul>
+            For the full syntax see [[[#abnf]]]
+        </p>
+    </section>
+    <section id="vocabulary">
+        <h2>PROFINET Vocabulary</h2>
+        This section describes the vocabulary used in the PROFINET protocol. 
+        A protocol binding template implementation should use the vocabulary defined in this section to
+        describe the different configuration that can be used to exchanged data between Web of Things.
+        <section>
+            <h3>URL terms</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Vocabulary term</th>
+                        <th>Description</th>
+                        <th>Assignment</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <!-- Generate forms terms from the ontology-->
+                    
+        
+        <tr>
+            <td><code>profv:slot</code></td>
+            <td>Specifies the location where the IO device is plugged into</td>
+            <td>required</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+
+        <tr>
+        
+
+        
+        <tr>
+            <td><code>profv:subslot</code></td>
+            <td>subslot provides the interface for the parameter to read/write, a slot can contain one or more subslots</td>
+            <td>required</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+
+        <tr>
+        
+
+        
+        <tr>
+            <td><code>profv:index</code></td>
+            <td>an identifier that uniquely points to a resource within slot/subslot combination of a PROFINET IO device</td>
+            <td>required</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+
+        <tr>
+
+        
+        <tr>
+            <td><code>profv:api</code></td>
+            <td>used to provide additional addressing to differentiates profiles parameter. if the term is not provided the TD processor defaults it to <code>zero</code></td>
+            <td>optional</td>
+            <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+
+        <tr>
+
+        <tr>
+            <td><code>profv:datalength</code></td>
+            <td>used for aplication and network optimization to appropriately define the specific data length of the resource payload.</td>
```

Comment:
```suggestion
            <td>used for application and network optimization to appropriately define the specific data length of the resource payload.</td>
```

Reply:
Including @danielpeintner's typo fix
```suggestion
            <td>Used in application and network optimization, to appropriately define the specific data length of the resource payload.</td>
```